OvmfPkg/AmdSev/AmdSevX64.dsc

```diff
@@ -545,6 +545,8 @@
 
   gEfiMdePkgTokenSpaceGuid.PcdFSBClock|1000000000
 
+  gUefiOvmfPkgTokenSpaceGuid.PcdOvmfFlashVariablesEnable|FALSE

```

Comment:
Why is this added everywhere?  FALSE is the default value.

Reply:
This PCD and the other changes around CC is mostly to work around the reliance the involved PCDs being dynamic.

Standalone MM does not support dynamic PCDs, thus if we want to stick to the PCD configurations from platform PEI and use those values at all, there are 3 options:
1. We need to use it as FixedAtBuild PCDs
2. Create hobs and pass along the information to Standalone MM
3. Hardcode the value and treat the hardcoded value as a design choice

For `PcdOvmfFlashVariablesEnable`, I think it makes sense to treat it as a requirement to have flash enabled if we are working with Standalone MM. But I am not sure about the CC related PCDs. Any suggestions?

Reply:
@kraxel Could you please let me know your thoughts on this? Thanks.

Reply:
SMM builds already require flash being preset.  There is an ASSERT in QemuFlashInitialize() which triggers if you try run a SMM build (PcdSmmSmramRequire=TRUE) without flash.  Yes, it makes sense for StandaloneMM to require flash too.  I don't think we have to change PcdOvmfFlashVariablesEnable to FixedAtBuild for that.  It makes sense for the StandaloneMM build to also set PcdSmmSmramRequire=TRUE.

For CC I'd suggest to simply declare the combination of CC + StandaloneMM as unsupported.

Reply:
I removed the PCD reference inside the Standalone MM FVB instance.

As per CC, I created a null library instance, and just use that library when Standalone MM is enabled.

Reply:
This is updated.